src/backgroundFetch.js

```diff
@@ -0,0 +1,167 @@
+import BackgroundFetch from 'react-native-background-fetch'
+import PushNotification from 'react-native-push-notification'
+import { AsyncStorage } from 'react-native'
+import moment from 'moment'
+import Config from './config/config'
+import { IS_LOGGED_IN } from './lib/constants/localStorage'
+import goodWallet from './lib/wallet/GoodWallet'
+import userStorage from './lib/gundb/UserStorage'
+
+const options = {
+  minimumFetchInterval: 15,
+  forceAlarmManager: false,
+  stopOnTerminate: false,
+  startOnBoot: true,
+  requiredNetworkType: BackgroundFetch.NETWORK_TYPE_ANY,
+  requiresCharging: false,
+  requiresDeviceIdle: false,
+  requiresBatteryNotLow: false,
+  requiresStorageNotLow: false,
+  enableHeadless: true,
+  periodic: true,
+}
+
+const task = async taskId => {
+  console.log('[BackgroundFetch] taskId: ', taskId)
```

Comment:
looks like most of the code here (e.g. connection check, wallet initialization) has been copy-pasted from the app.
this code should be moved to the separate functions and reused 
otherwise it will hard to support if, for example, we would like to change something in the initialization process